docs/user/ppl/functions/math.rst

```diff
@@ -569,23 +569,23 @@ Example::
     +----------+-----------------+------------------+
     | FLOOR(0) | FLOOR(50.00005) | FLOOR(-50.00005) |
     |----------+-----------------+------------------|
-    | 0        | 50              | -51              |
+    | 0        | 50.0            | -51.0            |
     +----------+-----------------+------------------+
 
     os> source=people | eval `FLOOR(3147483647.12345)` = FLOOR(3147483647.12345), `FLOOR(113147483647.12345)` = FLOOR(113147483647.12345), `FLOOR(3147483647.00001)` = FLOOR(3147483647.00001) | fields `FLOOR(3147483647.12345)`, `FLOOR(113147483647.12345)`, `FLOOR(3147483647.00001)`
     fetched rows / total rows = 1/1
     +-------------------------+---------------------------+-------------------------+
     | FLOOR(3147483647.12345) | FLOOR(113147483647.12345) | FLOOR(3147483647.00001) |
     |-------------------------+---------------------------+-------------------------|
-    | 3147483647              | 113147483647              | 3147483647              |
+    | 3147483647.0            | 113147483647.0            | 3147483647.0            |
     +-------------------------+---------------------------+-------------------------+
 
     os> source=people | eval `FLOOR(282474973688888.022)` = FLOOR(282474973688888.022), `FLOOR(9223372036854775807.022)` = FLOOR(9223372036854775807.022), `FLOOR(9223372036854775807.0000001)` = FLOOR(9223372036854775807.0000001) | fields `FLOOR(282474973688888.022)`, `FLOOR(9223372036854775807.022)`, `FLOOR(9223372036854775807.0000001)`
     fetched rows / total rows = 1/1
     +----------------------------+--------------------------------+------------------------------------+
     | FLOOR(282474973688888.022) | FLOOR(9223372036854775807.022) | FLOOR(9223372036854775807.0000001) |
     |----------------------------+--------------------------------+------------------------------------|
-    | 282474973688888            | 9223372036854775807            | 9223372036854775807                |
+    | 282474973688888.0          | 9.223372036854776e+18          | 9.223372036854776e+18              |
```

Comment:
For the double values which are out of the double's precision. Keep the behavior same as Java's `Math.floor(double)` and Splunk's `floor()` math function.